DEPLOYMENT.md

```diff
@@ -1,10 +1,12 @@
-# [DRAFT] Deployment of workshoppers
+# Deployment of workshoppers
 
-Suggesting steps to organize the process of publishing changes from workshoppers to npm.
+Below you can find details regarding the deployment process of [workshoppers on npm](https://www.npmjs.com/org/workshopper). 
```

Comment:
Sounds better - thanks. Please, feel free to apply further modifications to other phases too, I will not feel bad about it, and it will be easier and faster for you.